manual/src/html_processing/Makefile

```diff
@@ -50,24 +50,25 @@ $(WEBDIRCOMP)/%.js: js/%.js | $(WEBDIRCOMP)
 
 js: $(JS_FILES)
 
+CURL = curl -s
 # download images for local use
 SEARCH := search_icon.svg
 $(WEBDIRAPI)/search_icon.svg: | $(WEBDIRAPI)
-	curl "https://ocaml.org/img/search.svg" > $(WEBDIRAPI)/$(SEARCH)
+	$(CURL) "https://ocaml.org/img/search.svg" > $(WEBDIRAPI)/$(SEARCH)
```

Comment:
The curl change is orthogonal to the fix, isn't it?

Reply:
Yes - it was just noisy when I was diffing the commands, I so I quickly fixed it!